ldap/servers/plugins/replication/cl5_api.c

```diff
@@ -86,6 +87,12 @@
 #define NO_DISK_SPACE 1024
 #define MIN_DISK_SPACE 10485760 /* 10 MB */
 
+#define MAX_RETRIES 10 /* Maximum number of retry in case of db retryable error */
+#define CL5_TRIM_MAX_PER_TRANSACTION 100
+#define CL5_TRIM_MAX_LOOKUP_PER_TRANSACTION 10000
```

Comment:
This value seems high compare to the default db-lock. I was thinking that db can consume up to one lock per lookup page. Meaning that a single TRIM/PURGE loop can consume all the db-lock. Am I missing something ?

Reply:
1. That was the value in the old code    `2494             if (trimmed == 10000 || (batch_count && trimmed == batch_count)) {`
2. db_stat -h $DB -Cp shows that lock table is real size is nearly 16K (while 10K are configured) so 10K pages should be fine.)